src/cleed_io/cleed_io.c

```
@@ -0,0 +1,26 @@
+
+#include "cleed_io.h"
+
+int cleed_io_read_curve(const char *path, cleed_curve *out_curve)
+{
+  (void)path;
+  (void)out_curve;
+  errno = ENOSYS;
+  return -1;
+}
+
+int cleed_io_write_curve(const char *path, const cleed_curve *curve)
```

Cppcheck (2.19.0)

[style] The function 'cleed_io_write_curve' is never used.

(unusedFunction)